rinja_parser/src/lib.rs

```diff
@@ -635,14 +635,17 @@ impl<'a> State<'a> {
         }
     }
 
-    fn nest<'b>(&self, i: &'b str) -> ParseResult<'b, ()> {
+    fn nest<'b, T, F: FnMut(&'b str) -> ParseResult<'b, T>>(
+        &self,
+        i: &'b str,
+        mut callback: F,
```

Comment:
`F` can be `FnOnce`. Currently that does not matter, but maybe we refactor the parser code later.

Reply:
Good point!